### DISCUSSION TITLE
Use the MantineUI library for the user interface

### DISCUSSION THREAD

**Discussion by daawaan4x** [keystonejs/keystone#7488] [Feature requests]
From what I understand from keystone's repo, it seems like its using a custom UI library. Since we want the AdminUI to be both beautiful and customizable, I think its worth a shot to migrate to using MantineUI instead

**Comment by raveling** [comment]
@daawaan thanks for sharing this idea. Mantine UI looks very nice, but we intend to build a Keystone specific UI kit.
